src/main.ts

```diff
@@ -129,7 +129,7 @@ async function notifyTeams(notificationMessage: NotificationMessage) {
 }
 
 checkBasicInputsSet();
-if (core.getInput('job_status').toUpperCase() !== jobStatus.cancelled.status) {
+if (core.getInput('job_status') !== jobStatus.cancelled.status) {
```

Comment:
better, but it would be more resilient still to explicitly lower case the job status before comparing wouldn't it?

Reply:
I should have added a comment in the commit message for you to wait hehe! Based on the github documentation this should be lowercase and I wanted to test it out of curiosity, I have my playground project and tests
https://github.com/eleni-salamani/playground/runs/489814993?check_suite_focus=true
It seems they still return Cancelled so toLowerCase is needed for sure --> off to make the change and re-test.

Reply:
ok with the lowercase enforcement it works, see https://github.com/eleni-salamani/playground/runs/489829387
--> _##[warning]Job was cancelled, no notification will be sent_